src/Behat/Behat/Transformation/Context/Factory/TransformationCalleeFactory.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Behat.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Behat\Behat\Transformation\Context\Factory;
+
+use Behat\Behat\Transformation\Transformation;
+use Behat\Behat\Transformation\Transformation\ColumnBasedTableTransformation;
+use Behat\Behat\Transformation\Transformation\PatternTransformation;
+use Behat\Behat\Transformation\Transformation\ReturnTypeTransformation;
+use Behat\Behat\Transformation\Transformation\RowBasedTableTransformation;
+use Behat\Behat\Transformation\Transformation\TableRowTransformation;
+use Behat\Behat\Transformation\Transformation\TokenNameAndReturnTypeTransformation;
+use Behat\Behat\Transformation\Transformation\TokenNameTransformation;
+use ReflectionMethod;
+
+/**
+ * Generates the callee for a transformation
+ *
+ * @author Konstantin Kudryashov <ever.zet@gmail.com>
+ */
+class TransformationCalleeFactory
```

Comment:
I suggest marking this class a `@internal`. I don't think we want to make it covered by BC.